lib/command-palette-view.coffee

```diff
@@ -41,6 +55,10 @@ class CommandPaletteView extends SelectListView
       @show()
 
   show: ->
+    if @preserveLastSearch
+      @filterEditorView.setText(@lastSearch)
+      @filterEditorView.getModel().selectAll()
```

Comment:
It selects the text in the command palette, so if you want to use a different command then you don't need to select it first. Otherwise you can press enter and it will re-use the same command.